Update chef-workstation Hugo module

IanMadd opened the pull request:
Signed-off-by: IanMadd <imaddaus@chef.io>

### Description

Update the Hugo module for chef-workstation.
Merge on 6/2

### Definition of Done

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

Commit:
Update chef-workstation Hugo module

Signed-off-by: IanMadd <imaddaus@chef.io>

Commit:
Update chef-workstation Hugo module

Signed-off-by: IanMadd <imaddaus@chef.io>

tyler-ball committed:
Preparing Supported Versions page for Chef Workstation GA (#2503)

* Preparing Supported Versions page for Chef Workstation GA

Updating to show that ChefDK 3 is EOL and ChefDK 4 is deprecated

Signed-off-by: tyler-ball <tball@chef.io>

tas50 committed:
The 20.6 release is June not May (#2512)

Signed-off-by: Tim Smith <tsmith@chef.io>

mjingle reviewed (approved):
![tenor-179523007](https://user-images.githubusercontent.com/5807053/83553127-cb5d4500-a4d8-11ea-88fa-e2248fa12eab.gif)

schisamo committed:
Enable Chef Desktop documentation  (#2507)

* Netlify build should pull in Hugo modules via Git SSH

Currently you can only pull in private GitHub repos using SSH during
Netlify builds.

Signed-off-by: Seth Chisamore <schisamo@chef.io>

* Begin pulling in Chef Desktop docs

Signed-off-by: Seth Chisamore <schisamo@chef.io>

* Remove redundant hugo build in verify tests as we already do a preview build in Netlify

Signed-off-by: Seth Chisamore <schisamo@chef.io>

* update desktop hugo mod

Signed-off-by: IanMadd <imaddaus@chef.io>

* Add data and shortcodes

Signed-off-by: IanMadd <imaddaus@chef.io>

Co-authored-by: IanMadd <imaddaus@chef.io>
Co-authored-by: Ian Maddaus <IanMadd@users.noreply.github.com>

Commit:
Fix go.sum file

Signed-off-by: IanMadd <imaddaus@chef.io>

IanMadd committed:
Merge branch 'master' into im/update_workstation_docs
